tests/medium/test_android.py

```diff
@@ -54,7 +54,7 @@ class AndroidNDKContainer(ContainerTests, test_android.AndroidNDKTests):
 
     def setUp(self):
         self.hosts = {443: ["developer.android.com"]}
-        self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'android')
+        self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'android-ndk')
```

Comment:
I don't see you creating a separate android-ndk repository for medium tests but you did change that path, did you get the medium tests working with this change? Was there already a separate repository that I don't see? :)